include/odp/api/spec/time.h

```diff
@@ -76,6 +76,24 @@ odp_time_t odp_time_local(void);
  */
 odp_time_t odp_time_global(void);
 
+/**
+ * Current local time in nanoseconds
+ *
+ * Like odp_time_local(), but time stamp value is converted to nanoseconds.
+ *
+ * @return Local time stamp in nanoseconds
+ */
+uint64_t odp_time_local_ns(void);
```

Comment:
The current time API handles direct "tick" time and calculations on it. Abstract odp_time_t + functions provide portability since "tick" time may consist of multiple counters (like in Posix API sec + nsec).